packages/common/mui/src/lib/card-variants/incentive-leveraged/incentive-leveraged.tsx

```diff
@@ -31,21 +42,85 @@ interface ContentWrapperProps {
   hovered: boolean;
   theme: NotionalTheme;
 }
+interface CardValueProps {
+  title: ReactNode;
+  rate: string;
+  symbol: string;
+  incentiveData?: {
+    symbol: string;
+    incentiveAPY: number;
+  };
+  secondaryIncentiveData?: {
+    symbol: string;
+    incentiveAPY: number;
+  };
+  isIncentive?: boolean;
+}
+
+export const CardValue = ({
```

Comment:
@matthew-garrett this is the hover state, is that right? Can we rename the component to something that explains that? Like "IncentiveLeveragedHoverState"? Also this looks like it is exported, is it used elsewhere? If so then we should put it in its own file.